### PR TITLE
doc: fix rendering of multi-breadcrumbed groups

### PR DESCRIPTION
**Pull request by miri64**
Fixes the output of multi-breadcrumbed groups (groups that are in multiple groups) from something like e.g.

```
CPU » | NativeNetwork Device Drivers
```

to 

```
CPU » | Native Network Device Drivers
```

### CODE REVIEW

**Comment by aabadie** [comment]
To me the valid result should be:
```CPU >> Native | Device Drivers```

**Comment by smlng** [comment]
@miri64 can you provide a link where this fix will be applied to, I can't find any `NativeNetwork Device Drivers` ...

**Comment by miri64** [comment]
> To me the valid result should be:
CPU >> Native | Device Drivers

Not that easy, at least not with CSS3:

1. the `» |` is generated by Doxygen as text in the surrounding `.ingroups` `<div>`, so at least to my knowledge no way to remove them.
2. the `.el` links are not structured, they are just coming one after another (with the `» |` text in between if there is a hierarchy), so even if I can somehow remove the `» |` with CSS, I don't know how simple it then would be to return the proper breadcrumb hierarchy.
3. this might be possible, but *I* don't know how: I can't distinguish between the first and subsequent children of `div.ingroups`, so `| CPU » | Native | Device Drivers` would be possible, but not `CPU » | Native | Device Drivers`.

I could use JS, but that already had proven to need heavy rework after a simple Doxygen-Update on the doc.riot-os.org machine, so I'd like to stick to CSS when possible.

TL;DR I'm working with the things I'm given, I can't make wonders happen ;-).

> @miri64 can you provide a link where this fix will be applied to, I can't find any NativeNetwork Device Drivers ...

Any of the double `@ingroup`s in #7298 would generate something similar.

**Comment by aabadie** [comment]
and go!
